### ISSUE TITLE
Test/Train/Val split

### ISSUE THREAD

**Issue by DrJonnyT** [UoMResearchIT/machine-learning-neural-python#5]
dataset_train, dataset_temp, labels_train, labels_temp = train_test_split(dataset, labels, test_size=0.3, stratify=labels, random_state=42)
dataset_val, dataset_test, labels_val, labels_test = train_test_split(dataset_temp, labels_temp, test_size=0.5, stratify=labels_temp, random_state=42)

This makes it a 70:15:15 split. Also stratifies so you get a good mix of labels
